azure-pipelines-with-e2etests.yml

```diff
@@ -3,8 +3,8 @@
 # And then running E2E tests and publishing them.
 
 trigger:
-  - main
-  - develop
+  - 2.0-preview
+  - 'release*'
```

Comment:
I think we might want to be more specific than just "release*". I think we might want to use the prefix "release-2*" so that v1 releases aren't triggering these pipelines. Unless I'm overthinking this.

Reply:
Since these yml files are only in 2.0-preview branch, there's no need to further filter the trigger here. It doesn't exist in master, so it won't be triggered for v1 releases.